<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuDebug">
     <property name="title">
      <string>&amp;Debug</string>
     </property>
     <addaction name="actionDump_Tree_to_Console"/>
     <addaction name="actionDump_Backend_to_Console"/>
    </widget>
    <addaction name="actionConverter"/>
    <addaction name="actionSearch"/>
    <addaction name="action_Settings"/>
    <addaction name="separator"/>
    <addaction name="menuDebug"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Connect">
   <property name="text">
    <string>&amp;Connect...</string>
   </property>
  </action>
  <action name="action_Register">
   <property name="text">
    <string>&amp;Register...</string>
   </property>
  </action>
  <action name="action_Disconnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionPersons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Persons</string>
   </property>
  </action>
  <action name="actionPing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ping</string>
   </property>
  </action>
  <action name="actionprint_TabMaps">
   <property name="text">
    <string>print TabMaps</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About QAnnotate</string>
   </property>
  </action>
  <action name="actionDump_Tree_to_Console">
   <property name="text">
    <string>Dump &amp;Tree to Console</string>
   </property>
  </action>
  <action name="actionDump_Frontend_to_Console">
   <property name="text">
    <string>Dump Frontend to Console</string>
   </property>
  </action>
  <action name="actionDump_Backend_to_Console">
   <property name="text">
    <string>Dump &amp;Frontend to Console</string>
   </property>
  </action>
  <action name="actionConverter">
   <property name="text">
    <string>&amp;Converter</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
